### PR TITLE
chpass: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/osx/chpass.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+`chpass -s {{ruta/al/shell}}`
+
+- Establece un inicio de sesión [s]hell para un usuario específico:
+
+`chpass -s {{ruta/al/shell}} {{nombre_de_usuario}}`
+
+- Edita el registro de usuario en el nodo de directorio en la ubicación dada:
+
+`chpass -l {{ubicación}} -s {{ruta/al/shell}} {{nombre_de_usuario}}`
```
Comment (ikks) [inline pages.es/osx/chpass.md:22]:
```suggestion
`chpass -l {{ubicación}} -s {{ruta/al/shell}} {{usuario}}`
```

File: pages.es/osx/chpass.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+- Utiliza el nombre dado al autenticarse en el nodo de directorio que contiene al usuario:
+
+`chpass -u {{nombre_de_usuario}} -s {{ruta/al/shell}} {{nombre_de_usuario}}`
```
Comment (ikks) [inline pages.es/osx/chpass.md:26]:
```suggestion
`chpass -u {{usuario_autenticado}} -s {{ruta/al/shell}} {{usuario}}`
```

File: pages.es/osx/chpass.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+- Establece un inicio de sesión [s]hell para un usuario específico:
+
+`chpass -s {{ruta/al/shell}} {{nombre_de_usuario}}`
+
+- Edita el registro de usuario en el nodo de directorio en la ubicación dada:
+
+`chpass -l {{ubicación}} -s {{ruta/al/shell}} {{nombre_de_usuario}}`
+
+- Utiliza el nombre dado al autenticarse en el nodo de directorio que contiene al usuario:
```
Comment (ikks) [inline pages.es/osx/chpass.md:24]:
```suggestion
- Utiliza el [u]suario_autenticado al identificarse en el nodo de directorio que contiene a cierto usuario:
```

**Comment by ikks** [comment]
@kant I was wondering if we can use the word `usuario` for `username` in translations.  I have seen that usually the username is shorter than the real name.

I would be happy to make the necessary changes in [documentation](https://github.com/tldr-pages/tldr/blob/4d5dd59133e3a3b869e765acda7d874fafd4044b/contributing-guides/translation-templates/common-arguments.md?plain=1#L19) and also in previously translated files to unify as much as possible.

**Comment by kant** [comment]
@ikks roger that

**Review by ikks — APPROVED** [review]

**Review by karthik-script — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**